idaes/core/util/model_diagnostics.py

```diff
@@ -1052,6 +1080,168 @@ def display_near_parallel_variables(self, stream=None):
     # TODO: Block triangularization analysis
     # Number and size of blocks, polynomial degree of 1x1 blocks, simple pivot check of moderate sized sub-blocks?
 
+    def _collect_constraint_mismatches(self, descend_into=True):
+        walker = ConstraintTermAnalysisVisitor(
+            term_mismatch_tolerance=self.config.constraint_term_mismatch_tolerance,
```

Comment:
Even though this function is private, shouldn't we have some sort of description about what it does?